doc/cephfs/purge-queue.rst

```diff
@@ -0,0 +1,83 @@
+============
+Purge Queue
+============
+
+MDS maintains a data structure known as **Purge Queue** which is responsible
+for managing and executing the sequential deletion of files.
+There is a Purge queue for every MDS rank. Purge queues consist of purge items
```

Comment:
The first Purge here should be lowercased.

Reply:
There's a lot of stuff that needs to be rewritten here. I'm going to make a "purge queue" PR to hash out the purge queue definition, and I will also make a grammar-and-elegance pass. This suggestion will be incorporated into that latter PR. As always, good catch.